### PR TITLE
WTD: Projected volumes

### PR DESCRIPTION
**Pull request by jseldess**
- Update projected volume task:
   - Rename file
   - Build out steps
   - Create separate yaml file for pod configuration
   - Add new page to the sidenav
- Add projected volume type to volumes concept page

See https://github.com/kubernetes/kubernetes.github.io/pull/3771 for prior conversation and reviews.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3856)

### CODE REVIEW

**Comment by jseldess** [comment]
@jpeeler, @chenopis, my git setup was messy for https://github.com/kubernetes/kubernetes.github.io/pull/3771, so I closed it, set myself up better, and opened this PR with the relevant changes. Apologies for stretching this review across 2 PRs. 

Please let me know if there's anything else to do. Would love to merge this and move on to other issues.

File: docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+{% endcapture %}
+
+{% capture steps %}
+## Configure a projected volume for a pod
+
+In this exercise, you create username and password Secrets from local files. You then create a Pod that runs one Container, using a [`projected`](/docs/concepts/storage/volumes/#projected) Volume to mount the Secrets into the same shared directory.
+
+Here is the configuration file for the Pod:
+
+{% include code.html language="yaml" file="projected-volume.yaml" ghlink="/docs/tasks/configure-pod-container/projected-volumen.yaml" %}
```
Comment (jpeeler) [inline docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: looks like a misspelling of volume.yaml.

Comment (jseldess) [inline docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: Good catch. Done.

**Comment by jseldess** [comment]
@chenopis, I think this only needs your doc review (and it's the same as the previous PR).

**Comment by chenopis** [comment]
@jseldess Looks great. Thanks!

**Comment by jpeeler** [comment]
@mburke5678 ready for your integration efforts

**Pull request merged**